orchestration/dagster_orchestration/hca_orchestration/contrib/bigquery.py

```diff
@@ -1,25 +1,24 @@
-from typing import Optional
```

Comment:
Most (all?) of the changes in this service are to start returning `RowIterator` instances rather than `Job` instances. That is, rather than forcing the client to call `.result()` on the returned `Job` to actually do the thing (make the query do the extraction, whatever), the service will now do it and return the materialized results. 

This makes a cleaner interface and means less of a leaky abstraction (i.e., client code of this service should not have to know or care about dispatching jobs on BQ, and instead only have to care about the actual results).